dvc/fs/fsspec_wrapper.py

```diff
@@ -164,3 +149,36 @@ def _download(
             ) as wrapped:
                 with open(to_file, "wb") as fdest:
                     shutil.copyfileobj(wrapped, fdest, length=fobj.blocksize)
+
+
+# pylint: disable=abstract-method
+class CloudFSSpecWrapper(FSSpecWrapper):
```

Comment:
`Cloud` might be misleading (isn't `gdrive` in cloud?). How about `ObjectFSWrapper`? It is a bit of a mouthful though.

Reply:
What about PrefixBasedFSWrapper?

Reply:
@isidentical, I'd prefer `Object-*`, but no strong opinions.

Reply:
Then Let's go with ObjectFSWrapper. Thanks for the suggestion!